extensions/reactive-mssql-client/deployment/src/main/java/io/quarkus/reactive/mssql/client/deployment/ReactiveMSSQLClientProcessor.java

```diff
@@ -83,7 +84,16 @@ ServiceStartBuildItem build(BuildProducer<FeatureBuildItem> feature,
 
     @BuildStep
     DevServicesDatasourceConfigurationHandlerBuildItem devDbHandler() {
-        return DevServicesDatasourceConfigurationHandlerBuildItem.reactive(DatabaseKind.MSSQL);
+        return DevServicesDatasourceConfigurationHandlerBuildItem.reactive(DatabaseKind.MSSQL, new Function<String, String>() {
+            @Override
+            public String apply(String url) {
+                url = url.replaceFirst("jdbc:", "vertx-reactive:");
+                if (url.endsWith(";encrypt=false")) {
```

Comment:
Hmmm. But are you sure this parameter will always be at the end? That looks extremely brittle?

Reply:
Can't you just replace ";encrypt=false" with ""?

Reply:
Good point @gsmet. That should do @cescoffier